src/Common/src/Interop/Unix/libssl/SecuritySafeHandles.cs

```diff
@@ -56,12 +56,19 @@ protected override bool ReleaseHandle()
     // Implementation of handles dependable on FreeCredentialsHandle
     //
 #if DEBUG
-    internal sealed class SafeFreeCredentials : DebugSafeHandle
+    internal abstract class SafeFreeCredentials : DebugSafeHandle
```

Comment:
@bartonjs any suggestions on the folder that the base class needs to go into since there will be no real library corresponding to the base class?